pkgs/os-specific/bsd/freebsd/pkgs/fsck.nix

```diff
@@ -1,5 +1,34 @@
-{ mkDerivation }:
+{
+  lib,
+  mkDerivation,
+  libufs,
+  compatIfNeeded,
+  withUfs ? true,
+  withMsdosfs ? true,
+}:
```

Comment:
Is there a reason it makes more sense to combine them in 14.1, or is this something we should have done in 14.0?

Reply:
fsck is added in this PR, and while it's not related to 14.1 per se, the vm refused to boot until I added it.

Reply:
@artemist Perhaps you were asking about mount? But I have the same question in https://github.com/NixOS/nixpkgs/pull/330069/files#r1693173119